src/views/projectExplorer/index.ts

```diff
@@ -641,8 +647,37 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
 
           this.refresh();
         }
-      })
+      }),
+      commands.registerCommand(`vscode-ibmi-projectexplorer.createIProjError`, async (element: ErrorItem) => {
```

Comment:
Can simply the number of commands here by reusing the existing one. Using instanceOf